test/burnAuction.test.ts

```diff
@@ -1,18 +1,28 @@
 // Tests borrowed from https://github.com/iden3/rollup/blob/master/test/contracts/RollupBurnAuction.test.js
 import { assert, expect } from "chai";
 import { BigNumber, Signer } from "ethers";
-import { formatEther } from "ethers/lib/utils";
+import { formatEther, isAddress } from "ethers/lib/utils";
 import { ethers } from "hardhat";
 import { toWei } from "../ts/utils";
 import { MockRollup } from "../types/ethers-contracts/MockRollup";
 import { MockRollupFactory } from "../types/ethers-contracts/MockRollupFactory";
 import { TestBurnAuction } from "../types/ethers-contracts/TestBurnAuction";
 import { TestBurnAuctionFactory } from "../types/ethers-contracts/TestBurnAuctionFactory";
 
-const defaultAddress = "0x0000000000000000000000000000000000000000";
+const {
+    expectRevert // Assertions for transactions that should fail
```

Comment:
FYI we have expectRevert in the `./ts/utils.ts` too, don't need to add @openzeppelin/test-helpers dependency if that works.